src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -32405,31 +32409,33 @@ void F(bool b, object o1, object? o2, C<object> o3, C<object?> o4)
 " + NonNullTypesOff() + @"
     static C<object> o5 = null;
 }";
-            var comp = CreateCompilation(new[] { source });
+            var comp = CreateCompilation(new[] { source }, options: WithNonNullTypesTrue());
```

Comment:
>options: WithNonNullTypesTrue() [](start = 59, length = 31)

We probably should not change the tests. If these changes are needed for the tests to pass, please skip the tests instead, with a PROTOTYPE comment.